react-client/src/Components/App.js

```diff
@@ -5,16 +5,16 @@ import DomInfoDialogBox from "./DomInfoDialogBox";
 import { PRODUCTION_MODE } from "../keys";
 
 function App() {
-  // const [coordinates, setCoordinates] = useState({ top: 0, left: 0 });
+  const [notes, setNotes] = useState([]);
   const [domInfo, setDomInfo] = useState({
```

Comment:
My suggestion is you could've made whole `domInfo` state an array instead of managing single-value state to prevent you from manually changing the type of `coordinates` into an array for example.

Reply:
should i try changing it na ?? tapos same branch ko pa rin ipush?

Reply:
Yes. Logically it would make more sense na e array nalang ang DomInfo para di naka mag array sa mga properties nya.